org.gnome.Lollypop.json

```diff
@@ -0,0 +1,140 @@
+{
+    "app-id": "org.gnome.Lollypop",
+    "branch": "0.9.233",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.22",
```

Comment:
This was not working recently for me and as flatpak webpage is not updated, I guess it does not exist.